src/app/actions.ts

```diff
@@ -0,0 +1,13 @@
+'use server'
+
+import { createItem } from '@/lib/models/item'
+import { getCurrentAuthUser } from '@/lib/models/user'
+import { revalidatePath } from 'next/cache'
+
+export const addItemAction = async (formData: FormData): Promise<void> => {
+  const authUser = getCurrentAuthUser()
+  const name = formData.get('name') as string
+  const description = formData.get('description') as string
+  await createItem(authUser, name, description)
+  revalidatePath('/')
```

Comment:
When we mutate the item list, we want to revalidate the path that uses the item list (actually `/dashboard` would be more precise here, but `/` revalidates the entire app and this is a small app, so it works)